modular_skyrat/modules/loadouts/loadout_items/under/loadout_datum_under.dm

```diff
@@ -912,3 +912,12 @@ GLOBAL_LIST_INIT(loadout_miscunders, generate_loadout_items(/datum/loadout_item/
 /datum/loadout_item/under/formal/basil
 	name = "Basil Uniform"
 	item_path = /obj/item/clothing/under/costume/basil_boys
+
+/datum/loadout_item/under/formal/helltaker
+	name = "Demonic Suit"
+	item_path = /obj/item/clothing/under/rank/civilian/lawyer/beige/skirt/demon
+
+/datum/loadout_item/under/formal/helltakerbeeze
+	name = "Designer Demonic Suit"
+	item_path = /obj/item/clothing/under/rank/civilian/lawyer/beige/skirt/demon/beeze
```

Comment:
Same comment as for the other one, and we probably shouldn't put "helltaker" in the typepaths either.